src/Seq.Api/Model/Security/Permission.cs

```diff
@@ -46,10 +48,23 @@ public enum Permission
         /// Write-access to signals, alerts, preferences etc.
         /// </summary>
         Write,
-        
+
         /// <summary>
         /// Access to administrative features of Seq, management of other users, app installation, backups.
         /// </summary>
-        Setup,
+        [Obsolete("The `Setup` permission has been replaced by `Project` and `System`.")]
+        Setup = 5,
```

Comment:
Should probably drop off the integer representation here, as the API accepts strings for permission values.